examples/android/binder/java/io/grpc/binder/cpp/exampleserver/ExportedEndpointService.java

```diff
@@ -0,0 +1,28 @@
+package io.grpc.binder.cpp.exampleserver;
+
+import android.app.Service;
+import android.os.IBinder;
+import android.content.Intent;
+
+/** Exposes gRPC services running in the main process */
+public final class ExportedEndpointService extends Service {
+  private final IBinder binder;
+
+  static {
+    System.loadLibrary("app");
+  }
+
+  public ExportedEndpointService() {
+    init_grpc_server();
+    binder = get_endpoint_binder();
+  }
+
+  @Override
+  public IBinder onBind(Intent intent) {
+    return binder;
+  }
+
+  public native void init_grpc_server();
+
+  public native IBinder get_endpoint_binder();
```

Comment:
Will this helper class be part of the library or the users have to implement this themselves? If it is part of the library, should we put it to the `src/` folder instead?

Reply:
Yes you are right. I think eventually we want to do that, however at this moment I'm afraid that it will cause unnecessary confusion.

There is an "one artifact rule" that says one APK should only have single `.so` file containing native code. If we create a Java helper class under `src/` folder, then user will be responsible to make sure their `.so` file contains required symbols (i.e they need to manually depends on one of our `grpc_cc_library` in their `.so` target) and they correctly load the shared library before using our Java helper class.

For now I just let user to do all these work so it is clear that they need to handle these things by themselves